resources/src/jvmMain/kotlin/dev/icerock/moko/resources/desc/MokoBundle.kt

```diff
@@ -0,0 +1,48 @@
+package dev.icerock.moko.resources.desc
+
+import java.text.ChoiceFormat
+import java.text.Format
+import java.text.MessageFormat
+import java.util.*
+
+object MokoBundle {
+    private const val BUNDLE_NAME = "localization.MokoBundle"
+    private const val PLURALS_BUNDLE_NAME = "localization.MokoPluralsBundle"
+
+    fun getString(key: String, args: List<Any>? = null) =
+        with(ResourceBundle.getBundle(BUNDLE_NAME, Locale.getDefault()).getString(key)) {
+            if (args == null) {
+                this
+            } else {
+                MessageFormat.format(this, args)
+            }
+        }
+
+    /**
+     * This method works under the assumption, that the {0} argument is always the plural argument
+     * all others are simple arguments
+     * For ex: Lorem {1} ipsum {0} -> Is the correct format, if the second argument should be
+     * pluralized
+     */
```

Comment:
required tests for this part of logic...very hard to understand problem without cases